code/src/sixsq/nuvla/server/resources/nuvlabox.clj

```diff
@@ -1011,7 +1038,10 @@ particular NuvlaBox release.
                                      (and (a/can-manage? resource request)
                                           (nil? (:host-level-management-api-key resource))) (conj enable-host-mgmt-op)
                                      (and (a/can-manage? resource request)
-                                          (contains? resource :host-level-management-api-key)) (conj disable-host-mgmt-op))]
+                                          (contains? resource :host-level-management-api-key)) (conj disable-host-mgmt-op)
+                                     (and (a/can-manage? resource request)
```

Comment:
why not make this operation available only for `group/nuvlabox`. Since it should not be presented to the user. This will also  cancel this PR (Hide generate-new-api-key op from detail page [#768](https://github.com/nuvla/ui/pull/768))

Reply:
I think it should be presented to the user, once the menu bar has been fixed. #768 exists as a workaround, for now, to avoid overcrowding the menu bar